### PR TITLE
[DXP-1363] Adjust to new search endpoint

### PR DESCRIPTION
**Pull request by jgardo-ds**
## Description
Blueprint's endpoint exposing search was changed. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [ ] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code standards](/CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

**Comment by jgardo-ds** [comment]
I have read the Dynamic Solutions Individual Contributor License Agreement (CLA) Document and I hereby sign the CLA

**Comment by jgardo-ds** [comment]
recheck

**Review by marekczajkowski — APPROVED** [review]

**Review by pawel-boguski-ds — APPROVED** [review]

File: applications/puresight/frontend/src/components/Search/service.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ const SEARCH_RESULTS_COUNT = 40;
 const PREDEFINED_RESULTS_COUNT = 2;
 
 const buildUrl = (query: string, limit: number) => {
-  return `/search/byQuery?size=${limit}&query=${query}*`;
+  return `/search/query?size=${limit}&query=${query}`;
```
Comment (tomaszmichalak) [inline applications/puresight/frontend/src/components/Search/service.ts:49]: I like `query` ;)

**Review by tomaszmichalak — APPROVED** [review]
You can check my accesses to this repo :)

**Comment by hamster-bay** [comment]
> You can check my accesses to this repo :)

@tomaszmichalak thank you for taking the time to review and comment on our PR—it’s great to see you’re still interested in what we’re working on! 😊

Just to clarify, this is a public repository, so anyone can join in on discussions, reviews, or contributions. We’d love to see your input on our other public projects as well—perhaps you’ve had a chance to check out our latest [StreamX example](https://github.com/streamx-hub/streamx-examples) repository?

Cheers,
Kasia

**Pull request merged**